esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
```

Comment:
Should this be "/esp/logout"?  Given its on the frame its probably ok I suspect.